web/assets/toggle-insert-checkboxes.js

```diff
@@ -0,0 +1,23 @@
+const toggleInsertCheckboxes = () => {
+    /*Insert checkboxes may appear twice and therefore js is needed to make sure
+    that a check in one tab is displayed in the other*/
+
```

Comment:
A visual illustration of what this is solving, in case this comment doesn't make sense 
<img width="935" alt="Screenshot 2022-12-21 at 13 38 58" src="https://user-images.githubusercontent.com/51047911/208918574-6b802d60-0e8d-426d-a907-426b050c21be.png">
<img width="935" alt="Screenshot 2022-12-21 at 13 39 09" src="https://user-images.githubusercontent.com/51047911/208918552-f59a535a-0cc8-468c-9604-af3aa1c00a98.png">

Reply:
I tried to look displaying the same checkbox twice but as tab content sits in different containers to one another, they need to be separate checkboxes